build.js

```diff
@@ -135,13 +135,19 @@ function build() {
     ))
 
     .use(redirect({
-      '/download/swarmtool': 'http://repo2.maven.org/maven2/org/wildfly/swarm/swarmtool/'+ versions.CURRENT_RELEASE + '/swarmtool-' + versions.CURRENT_RELEASE + '-standalone.jar',
-      '/download/microprofile-hollowswarm': 'http://repo2.maven.org/maven2/org/wildfly/swarm/servers/microprofile/' + versions.CURRENT_RELEASE + '/microprofile-' + versions.CURRENT_RELEASE + '-hollowswarm.jar',
-      '/download/microprofile-jpa-hollowswarm': 'http://repo2.maven.org/maven2/org/wildfly/swarm/servers/microprofile-jpa/' + versions.CURRENT_RELEASE + '/microprofile-jpa-' + versions.CURRENT_RELEASE + '-hollowswarm.jar',
-      '/download/keycloak-swarm': 'http://repo2.maven.org/maven2/org/wildfly/swarm/servers/keycloak/' + versions.CURRENT_RELEASE + '/keycloak-' + versions.CURRENT_RELEASE + '-swarm.jar',
-      '/download/management-console-swarm': 'http://repo2.maven.org/maven2/org/wildfly/swarm/servers/management-console/' + versions.CURRENT_RELEASE + '/management-console-' + versions.CURRENT_RELEASE + '-swarm.jar',
-      '/download/swagger-ui-swarm': 'http://repo2.maven.org/maven2/org/wildfly/swarm/servers/swagger-ui/' + versions.CURRENT_RELEASE + '/swagger-ui-' + versions.CURRENT_RELEASE + '-swarm.jar',
-      '/download/web-swarm': 'http://repo2.maven.org/maven2/org/wildfly/swarm/servers/web/' + versions.CURRENT_RELEASE + '/web-' + versions.CURRENT_RELEASE + '-hollowswarm.jar',
+      '/download/swarmtool': 'http://repo2.maven.org/maven2/io/thorntail/swarmtool/'+ versions.CURRENT_RELEASE + '/swarmtool-' + versions.CURRENT_RELEASE + '-standalone.jar',
+      '/download/microprofile-hollow-thorntail': 'http://repo2.maven.org/maven2/io/thorntail/servers/microprofile/' + versions.CURRENT_RELEASE + '/microprofile-' + versions.CURRENT_RELEASE + '-hollow-thorntail.jar',
```

Comment:
Should the "-swarm" ones be removed?

Reply:
I think we kept them to make sure old links/bookmarks still work. If the only links are from `src/downloads.adoc` and `src/index.adoc`, then we can remove them.